misk-sqldelight-testing/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+plugins {
+  kotlin("jvm")
+  `java-library`
+  id("app.cash.sqldelight")
```

Comment:
I didn't add the `id("com.vanniktech.maven.publish.base")` plugin here - I think that means this gradle module shouldn't publish to maven but would be nice to get confirmation. My intention is for this package to be internal-only.